### PR TITLE
Migrating to build.sbt and play-bootstrap-25

### PR DESCRIPTION
**Pull request by chrisjameswright**

### CODE REVIEW

**Comment by TheNickWilson** [comment]
LGTM

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.17
```
Comment (chriswaugh) [inline project/build.properties]: This line is breaking everything for me. As far as I can see version 0.13.18 doesn't exist? https://www.scala-sbt.org/download.html

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
...
+sbt.version=0.13.18
```
Comment (chriswaugh) [inline project/build.properties]: @chrisjameswright This line is breaking everything for me. As far as I can see version 0.13.18 doesn't exist? https://www.scala-sbt.org/download.html

**Comment by chriswaugh** [comment]
LGTM too :+1:

**Pull request merged**